.evergreen/config_generator/components/funcs/install_uv.py

```diff
@@ -18,16 +20,20 @@ class InstallUV(Function):
                     exit 1
                 fi
 
-                uv_install_dir="${MONGO_CXX_DRIVER_CACHE_DIR}/uv-0.5.9"
+                uv_install_dir="${MONGO_CXX_DRIVER_CACHE_DIR}/uv-0.5.14"
                 mkdir -p "$uv_install_dir"
 
-                if ! command -V "$uv_install_dir/uv" 2>/dev/null; then
+                if ! command -v "$uv_install_dir/uv" 2>/dev/null; then
                     env \\
                         UV_INSTALL_DIR="$uv_install_dir" \\
-                        UV_NO_MODIFY_PATH=1 \\
-                        mongo-cxx-driver/.evergreen/scripts/uv-installer.sh
+                        UV_UNMANAGED_INSTALL=1 \\
+                        INSTALLER_PRINT_VERBOSE=1 \\
+                        mongo-cxx-driver/.evergreen/scripts/uv-installer.sh --verbose
```

Comment:
```suggestion
                        mongo-cxx-driver/.evergreen/scripts/uv-installer.sh
```

Appears redundant with `INSTALLER_PRINT_VERBOSE`.